### PR TITLE
Correct `ProposalCount` & `Approvals`

### PR DESCRIPTION
**Pull request by AurevoirXavier**
Companion for Darwinia: https://github.com/darwinia-network/darwinia/pull/710

### CODE REVIEW

File: bin/node/runtime/pangolin/src/lib.rs [hunk 1]
```diff
@@ -949,7 +957,13 @@ fn migrate_treasury() {
...
 
+			ring_proposals_count += 1;
+
 			migration::put_storage_value(NEW_PREFIX, b"Proposals", &hash, new_proposal);
+
+			if approvals.contains(&index) {
+				ring_approvals.push(index);
+			}
 		}
 		if old_proposal.kton_value != 0 {
```
Comment (AsceticBear) [inline bin/node/runtime/pangolin/src/lib.rs:968]: Do we need to check `ring_bond` field?

Comment (AurevoirXavier) [reply bin/node/runtime/pangolin/src/lib.rs:968]: Why

Comment (AsceticBear) [reply bin/node/runtime/pangolin/src/lib.rs:968]: O, typo mistake. I want to say `kton_bond`. I am not sure if there are proposals kton_value = 0, kton_bond !=0.

Comment (AurevoirXavier) [reply bin/node/runtime/pangolin/src/lib.rs:968]: Check value is okay.

**Review by AsceticBear — APPROVED** [review]

**Review by HackFisher — APPROVED** [review]

**Pull request merged**